web_app/dist/main.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="icon" type="images/png" href="images/icon.png" />       
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>TaskViewer</title>
+    <meta name="description" content="Schedule tasks, without the clutter">
+    <link rel="stylesheet" href="main.css">   
+</head>
+ 
+<body>    
+    <div id="sticky-header">
+        <div class="navbar">
+            <img id="logo" src="images/icon_small.png" alt="TaskViewer Logo">
+            <a class="hover-transition">Log out</a>
+            <a class="hover-transition">Sort</a>
+            <a class="hover-transition">Filter by</a>
+            <div id="search-bar-container-anchor-right">
+                <input id="search-bar" type="text" placeholder="Search...">
+            </div>
+        </div>
+        <div class="new-flex-row-anchor-right">
+            <button id="create-task-button" class="button hover-transition">+</button>    
+        </div>
+        <div class="new-flex-row-anchor-right">
```

Comment:
The old comment got cleared out by another change, we should only need one `new-flex-row-anchor-right` div that contains both buttons, you might need to change the CSS a bit to make that work